src/test/resources/properties/application.properties

```diff
@@ -0,0 +1,3 @@
+//env=test
```

Comment:
I suppose this line was commented for passing of CI, but it would be great to see such a description in code, what do you think?

Reply:
actually good idea will change it